src/app/shared/services/title.service.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from '@angular/core';
+import {Title} from '@angular/platform-browser';
+
+@Injectable()
+export class TitleService {
+
+  constructor(private titleService: Title) {
+  }
+
+  readonly constTitle = 'Nike';
+
+  setTitleWithConstant(newTitle: string) {
+    this.titleService.setTitle(this.constTitle + ' ' + newTitle);
```

Comment:
`
let names = [this.constTitle];
if ( newTitle ) names.push(newTitle);
this.titleService.setTitle(names.join(': '))`